### ISSUE TITLE
issue with  labels.jsonl 

### ISSUE THREAD

**Issue by Niru864** [ZN1010/PEaCE#1]
Thank you for sharing your work and providing the PEaCE dataset. I'm currently working on fine-tuning an OCR model using your dataset and encountered some issues with the labels.jsonl file. Specifically, I'm running into multiple improperly formatted lines that the JSON parser cannot decode correctly.

For example, here are some of the errors I received:

```
Skipping invalid JSON line: {'render_1199231.jpg': '1 4 , 9 3 1'}
Skipping invalid JSON line: {'render_1199232.jpg': '4 9 , 3 9 0 + 1 8 , 7 9 3 / 3 7 , 8 2 1'}
Skipping invalid JSON line: {'render_1199233.jpg': '4 , 8 3 7 . 1 6 2'}
```
It seems that many lines are not properly formatted for JSON, with single quotes instead of double quotes or other inconsistencies. I'm currently trying to preprocess the data, but a significant number of these lines are being skipped due to the formatting issues, which affects the quality and completeness of the training data I have available.

I wanted to reach out to see if:

This is a known issue with the dataset.
There is an updated version of labels.jsonl that might address these formatting problems.
You have any suggestions on how I might proceed to resolve these issues or if you recommend any specific changes to how I am processing the dataset.

**Comment by c-heat16** [comment]
How are you trying to read the labels? We read the labels line-by-line similar to as described below and did not have these errors. Can you please try that and see if it fixes the issue?

```python
with open(labels_fp, 'r') as f:
    for line in f:
        line_j = eval(line)
```
